src/components/Register.module.css

```diff
@@ -0,0 +1,39 @@
+.headerBackground {
+  background-image: url(../assets/LAPalmTrees.png);
+  height: 250px;
+  background-size:cover;
+  min-width: 100%;
+}
+
+.card {
+  position: relative;
+}
+
+.h1 {
+  text-align: center;
+}
```

Comment:
I would probably say name it something else but otherwise not a big deal.

Reply:
done